GitUI/Translation/Czech.xlf

```diff
@@ -6456,7 +6456,7 @@ Would you like to do it now?</source>
         
       </trans-unit>
       <trans-unit id="_pullRepositoryButtons.Text">
-        <source>Pull with the default pull action ({0})|Pull with rebase|Pull with merge|Force push|Cancel</source>
+        <source>Pull with the default pull action ({0})|Pull with rebase|Pull with merge|Force push with lease|Cancel</source>
         
       </trans-unit>
       <trans-unit id="_pullRepositoryCaption.Text">
```

Comment:
Only the English.xlf needs to updated and committed (using the TranslateApp.exe).
The translations are updated when a release is built.

Reply:
Should I revert other files or it does not make sense because they will be updated automatically?

Reply:
I'd revert them.